api/AudioContext.json

```diff
@@ -0,0 +1,512 @@
+{
+  "api": {
+    "AudioContext": {
```

Comment:
I've open the issue #472 to define what we will do with mixins. It is fine to skip them for the moment.